src/components/Card.js

```diff
@@ -36,15 +36,15 @@ class Card {
 
   handleDeleteSubmit() {
```

Comment:
Needs correction: please remove this unused method

Reply:
I removed the "handleDeleteSubmit" unused method.